packages/stores/src/queries/pool-incentives/incentivized-pools.ts

```diff
@@ -237,6 +335,7 @@ export class ObservableQueryIncentivizedPools extends ObservableChainQuery<Incen
                   mintPrice.toString()
                 ).mul(yearProvisionToPot.toDec());
 
+                // yearProvisionToPotPrice / tVL
```

Comment:
Can we please remove all comments in this existing function? And otherwise any comments that don't contribute to code comprehension